src/docker-images/RepairManagerAgent/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:18.04
```

Comment:
Maybe you can use `python:3.7` base image, that will make repair manager agent image much smaller. For ref https://github.com/microsoft/DLWorkspace/blob/dltsdev/src/docker-images/watchdog/Dockerfile#L18